Assets/Scripts/GraphService.cs

```diff
@@ -175,22 +176,22 @@ void FixEdgeGroup(IList<Edge> edges, Node nodeFrom, Node nodeTo)
     {
       var startingPosition = nodeFrom.Position;
       var endingPosition = nodeTo.Position;
-
-      var linePositions = lineFactory.GetLinePositionsFor(startingPosition, endingPosition, edges.Count);
-      for (var i = 0; i < edges.Count; i++)
+      for (int i = 0; i < edges.Count; i++)
       {
-        edges[i].segmentGroup.PlaceAlongPoints(linePositions[i]);
+        var rotation = edges.Count == 1 ? null as float? : i * (360f / edges.Count);
+        var intermediatePoints = lineFactory.GetIntermediatePoints(startingPosition, endingPosition, rotation);
```

Comment:
I think that it would be simpler and clearer if lineFactory accepted 0 for straight edge instead of null. It's both true and simple and this could be i * (360f / edges.Count) % 360f

Reply:
It wouldn't work right now as rotation = 0 will curve the line (it will be the first curved line for the edge group). Null will generate a straight line. 
I think the best solution is to divide the GetIntermediatePoints method to curved and straight